de.json

```diff
@@ -692,19 +692,19 @@
 		"toggle-edit": "Umschalten zwischen Editor/Lesemodus",
 		"toggle-source-mode": "Umschalten zwischen Live-Vorschau/Quellcode-Ansicht",
 		"delete-current-file": "Aktuelle Datei löschen",
-		"new-tab": "Neuer Tab",
-		"close-all-tabs": "Close all tabs",
-		"close-active-tab": "Aktuellen Tab schließen",
-		"close-other-tabs": "Alle anderen Tabs schließen",
+		"new-tab": "Neues Tab",
+		"close-active-tab": "Schließe aktuelles Tab",
```

Comment:
Die alte Übersetzung ließt sich besser.
Auch in anderer Software Dokumentation (etwa Chrome & Firefox) wird "neuer/aktuellen" verwendet.

Reply:
ok. also eher die männliche Schreibweise.
Du hast Recht. Ich habe mir die anderen Übersetzungen zu Tab-Themen in Obsidian angeschaut. Dort ist es in der Tat die männliche Schreibweise. 
Danke für den Hinweis.